.ci/oci.Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
 
-FROM registry.ci.openshift.org/openshift/release:golang-1.16
+FROM registry.ci.openshift.org/openshift/release:golang-1.18
```

Comment:
asking the dumb questions today... 

could you use this instead?

```
# https://registry.access.redhat.com/ubi8/go-toolset
FROM registry.access.redhat.com/ubi8/go-toolset:1.18.4-8 as builder
```

Reply:
Potentially? I've stuck with the ci image since I assumed it had required tooling for ancillary prow test details. @musienko-maxim would likely be the person to ask.